src/_data/var.yml

```diff
@@ -19,8 +19,16 @@ b2b: Magento Commerce for B2B
 mbi: Magento Business Intelligence
 
 # Cloud product name variables
+
 ece: Magento Commerce Cloud
+csuite: Magento Commerce Cloud Suite
 ct: ece-tools
+mcp-prod: Magento Cloud Patches
+mcp: magento-cloud-patches
```

Comment:
I doubt the repositories will change names much, it is mostly for the product names. I get what you are trying to do, though.